src/app/components/layout/PageContent.tsx

```diff
@@ -5,14 +5,30 @@ import { useBackgroundColor } from '../../hooks/useBackgroundColor';
 export type PageContentProps = HTMLAttributes<HTMLDivElement> & {
     noPadding?: boolean;
     backgroundColor?: string;
+    wideLayout?: boolean;
+};
+
+const PAGE_WIDTH = {
+    WIDE: 1024,
```

Comment:
the 676 feels quite narrow on a standard 1920 monitor (I can only imagine the ultrawide ;))
It also feels a little strange having the content width jump around while navigating from page to page.
Can we compromise on a single width for everything? Maybe 900?

Reply:
I tried to see if I can make the text part to be 676px wide but the images/components to be 1024 wide. But since we are on `.mdx`, anything inside `<span>` is interpreted differently compared to just pure `.md`. 

> Can we compromise on a single width for everything? Maybe 900?

Bargain: at most 766px, stolen from material.io.

Reply:
Something like this:
![image](https://user-images.githubusercontent.com/29152776/80971255-7996a180-8dea-11ea-8eec-595304889f77.png)